client/src/cardComponent/cardComponent.css

```diff
@@ -0,0 +1,38 @@
+* {
```

Comment:
using `*` in any component's CSS can be dangerous. You only want these properties to reflect in the relevant component. use`#main` instead.